.github/workflows/ci-workflow.yml

```diff
@@ -36,6 +36,7 @@ jobs:
           name: nanox
         - path: $NANOSP_SDK
           name: nanosp
+        # add stax
```

Comment:
I would suggest that we create an issue in the tracker for this and change the comment to `# TODO(issue-xx): add stax`. This format makes it possible to search for outstanding TODOs while ensuring that we have recorded these tasks outside of the code.

I will leave it up to you whether we should adopt this approach or not.

Reply:
My goal is not to merge this PR before this is addressed. So this todo should be removed.
I suggest we keep this discussion unresolved so that we don't forget it :)